sdk/storage/Azure.Storage.DataMovement/tests/Shared/StartTransferUploadDirectoryTestBase.cs

```diff
@@ -162,7 +162,7 @@ private async Task UploadDirectoryAndVerifyAsync(
 
         [RecordedTest]
         [TestCase(Constants.KB, 10)]
-        [TestCase(12345, 10)]
+        [TestCase(Constants.KB * 10, 10)]
```

Comment:
Made this change for Page blobs. It is much easier to make this change then pass in some sort of flag for page blobs and assert exception, etc

Reply:
See my earlier comment about asserting inconclusive in pageblob impl. We probably want this test of an odd size. Let blobs that can support it have it.

Reply:
@jaschrep-msft let me know what you think